Makefile

```diff
@@ -13,6 +13,8 @@ ifneq (,$(DAPPER_HOST_ARCH))
 
 # Running in Dapper
 
+include Makefile.inc
```

Comment:
If we need this here, line 26 should be removed (but I’m not sure why it would make a difference).

Reply:
Ah my bad.... And I think it makes the difference since currently we decide on the settings yaml (OVN `.shipyard.e2e.ovn.yml` / Standard `$(DAPPER_SOURCE)/.shipyard.e2e.yml`)  BEFORE we actually import the value of `using` from `Makefile.inc`....

Reply:
Ah yes, I missed the reference to `_using` just below. So we need this line here, and the `include` further down removed.

Reply:
Fixed!